85.md

```diff
@@ -0,0 +1,70 @@
+NIP-85
+======
+
+Reviews
+-------
+
+This NIP describes several different types of `review` which follow a single set of conventions.
+
+In all cases, the `content` field SHOULD include a human-readable review. A review event's `d` tag indicates the object of the review.
+
+# Ratings
+
+Tags MAY contain additional `rating` tags each of which should have a `value` between 0 and 1,
+and an optional `mark` specifying an attribute of the thing being reviewed.
+
+# Review Kinds
+
+## Event
+
+Kind `31986` indicates a review of any event generated by a `pubkey`.
+
+- The `d` tag MUST be the `id` of the event.
+- The `p` tag MUST be the `pubkey` of the event.
+
+```json
+{
+  "kind": 31986,
+  "content": "Always publishing quality information about breaking news all over the world.",
+  ...
+  "tags": [
+    ["d", "<event-id>"]
+    ["p", "<pubkey>"],
+    ["k", "1"],
+    ["rating", "0.8"]
+  ],
+}
+```
+
+## Relays
+
+Kind `31987` indicates a review of a relay. The `d` tag MUST be the url of the relay.
+
+```json
+{
+  "kind": 31987,
+  "content": "This relay is fast!",
+  "tags": [
+    ["d", "wss://relay.example.com/"],
```

Comment:
Imo it should be namespaced otherwise there could be collisions with urls of other protocols that use websockets

Reply:
It's already namespaced, that's what kinds are for

Reply:
Lol, my bad got confused between nips